packages/frontend/core/src/mobile/pages/workspace/detail/mobile-detail-page.tsx

```diff
@@ -237,6 +247,12 @@ const MobileDetailPage = ({
     },
     [openJournal]
   );
+
+  useGlobalEvent(
+    'scroll',
```

Comment:
shall we add `passive` mode for events like `scroll`?

Reply:
Seems not matter? As mentioned on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#using_passive_listeners):

> You don't need to worry about the value of `passive` for the basic [scroll](https://developer.mozilla.org/en-US/docs/Web/API/Element/scroll_event) event. Since it can't be canceled, event listeners can't block page rendering anyway.

And other events like `touchstart`, `touchmove` should already be optimized by browser.

Reply:
I see, thanks!